components/Pagination/Pagination.js

```diff
@@ -10,19 +10,40 @@ Pagination.propTypes = {
   className: string,
   currentPage: number.isRequired,
   totalPages: number.isRequired,
+  maxElements: number,
 };
 
 Pagination.defaultProps = {
   className: undefined,
+  maxElements: 11,
 };
 
 // eslint-disable-next-line react/prop-types
-const PaginationItems = ({ currentPage, totalPages }) => {
-  const shouldTruncateStart = currentPage - 1 > 5;
-  const shouldTruncateEnd = totalPages - currentPage > 5;
+const PaginationItems = ({ currentPage, totalPages, maxLength }) => {
+  // maxElements is maximum length of the Pagination Bar, should be an odd integer, delault is 11
+  const maxElements = maxLength % 2 === 0 ? maxLength - 1 : maxLength;
+  const elementsOnOneSide = (maxElements - 1) / 2;
```

Comment:
Then we can just hard-code this as a constant

`const MAX_CLICKABLE_VISIBLE = 7`

Reply:
Yeah it depends on `maxElements` and as we are dropping it, we can hard code this.